src/commands/distribute/release.ts

```diff
@@ -268,7 +269,46 @@ export default class ReleaseBinaryCommand extends AppCommand {
         throw failure(ErrorCodes.Exception, `failed to set distribution group and release notes for release ${releaseId}`);
       }
     }
+  }
+
+  private async getDistributionGroup(client: AppCenterClient, releaseId: number, destination: string): Promise<models.DistributionGroupResponse> {
```

Comment:
Similar code is used in `add-destination.ts` - I suggest we refactor this in to a help file along with any other functions for accessing the distribution groups api. I guess this is made harder by the fork/PR stuff we're having to use right now - I'm guessing it might be easier to merge all this and then have a refactor PR that sort it out?

Reply:
do we have any existing pattern for code sharing across commands?

Reply:
Yes - if you make sure it's in a `lib` folder it shouldn't get picked up by the command / help creation routines.

Cross command I would put it in:
src/commands/distribute/lib

Our convention seems to be using either `*-helper.ts` or `*-util.ts`

Reply:
You can also share amongst beacons by putting the file in /src/commands/lib - but I don't think you need to do that here.